src/lib/pages/nexus-endpoint.svelte

```diff
@@ -17,17 +17,21 @@
   let {
     endpoint,
     editDisabled = false,
+    editHidden = false,
+    backHref = routeForNexus,
     taskQueueStatus,
   }: {
     endpoint: Endpoint;
+    backHref?: () => string;
```

Comment:
Let's make this a string not a function.

So default prop would be

`backHref = routeForNexus()`